### PR TITLE
Update test inputs

### PR DESCRIPTION
**Pull request by astolman**
I have added a couple of test inputs to the tar on AWS for unit tests in an upcoming pull request in the enterprise repo. This updates the test inputs from v26 to v27.

### CODE REVIEW

File: test/install_conda_packages_test.Dockerfile [hunk 1]
```diff
@@ -38,7 +38,7 @@ COPY packages /packages
 
 FROM pre_install AS test_python
 ARG CONDA_CLEAN="conda clean --quiet --yes --all"
-ARG MAMBA_INSTALL="mamba install --quiet --yes --channel katanagraph --channel /packages"
+ARG MAMBA_INSTALL="mamba install --quiet --yes --channel katanagraph --channel /packages -c katanagraph/label/test -c defaults"
```
Comment (arthurp) [inline test/install_conda_packages_test.Dockerfile]: I don't want to include defaults here because in the past I have had confusing problems when mixing anaconda and conda-forge packages. Why did you add this? I'd like to try to figure out another solution for you if possible.

**Review by arthurp — CHANGES_REQUESTED** [review]
The change of test version is great. I'm worried about the change to the conda channels. Let's talk about it.

File: test/install_conda_packages_test.Dockerfile (hunk 1 above)
Comment (astolman) [reply test/install_conda_packages_test.Dockerfile]: This actually must have come from jason's branch which I was tracking. I don't actually know if it is necessary.

**Comment by astolman** [comment]
Okay, I managed to rebase onto master, only the changes I need right now are in this PR.

**Review by arthurp — APPROVED** [review]
OK. This looks much better. :-)

~~BTW, if you want a rereview you need to click the rereview icon next to my name. I don't watch messages on PRs well, but the rereview request will get added to my TODO list.~~

**Comment by arthurp** [comment]
Actually, you did request a rereview. Sorry. My bad. You were on top of this, I was not.
